central/complianceoperator/v2/compliancemanager/manager.go

```diff
@@ -25,9 +25,12 @@ type Manager interface {
 	// HandleScanRequestResponse processes response of compliance scan configuration from a sensor.
 	HandleScanRequestResponse(ctx context.Context, requestID string, clusterID string, responsePayload string) error
 
+	// HandleDeleteScanRequestResponse processes response of delete compliance scan configuration from a sensor.
+	HandleDeleteScanRequestResponse(ctx context.Context, requestID string, clusterID string, responsePayload string) error
```

Comment:
Can we change responsePayload to response error? If there is a responseError, then that indicates that something went wrong